### ISSUE TITLE
Jack told me to do this

### ISSUE THREAD

**Issue by 1point618** [keybase/client#1852]
Keybase GUI Version: 1.0.9-20160130013621+d245df9

<img width="575" alt="screen shot 2016-01-29 at 10 19 22 pm" src="https://cloud.githubusercontent.com/assets/876894/12693387/72cdd844-c6d6-11e5-8d8f-c9e830f4b403.png">

**Comment by 1point618** [comment]
<img width="558" alt="screen shot 2016-01-29 at 10 20 45 pm" src="https://cloud.githubusercontent.com/assets/876894/12693391/900a5392-c6d6-11e5-8656-523700644b32.png">

**Comment by maxtaco** [comment]
Thanks for the bug report.  Did you previously installed keybase via brew?

**Comment by 1point618** [comment]
Hi @maxtaco, I had not no. This is my first time installing. @oconnor663 and I just created the account.

**Comment by maxtaco** [comment]
OK, thanks for the bug report.  We'll look into it ASAP

**Comment by 1point618** [comment]
Np!

**Comment by maxtaco** [comment]
Is it possible you were running `login` in two different windows?

**Comment by 1point618** [comment]
I don't think so. I only had the one terminal window open. I had just installed keybase moments before, so I'd assumed it had something to do with that.

**Comment by maxtaco** [comment]
ok thanks, we'll continue to scratch our heads!

**Comment by 1point618** [comment]
OK no worries, got it working anyway so I'm happy at least :). LMK if I can be helpful though!
